underdocs-renderer/src/main/kotlin/underdocs/renderer/output/html/render/section/SectionRenderer.kt

```diff
@@ -310,31 +311,49 @@ class SectionRenderer(private val linker: Linker) {
             h2("See Also")
     ).withClass("see-also")
 
-    fun renderReturnValue(visitable: Visitable, returnValue: String): Tag<*> {
-        val elements = ArrayList<DomContent>()
-
-        elements.add(h2("Return Value"))
-
-        if (visitable is Function) {
-            elements.add(p(
-                    code(referredTypeToString(visitable.returnType))
-            ).withClass("return-type"))
-        }
-
-        elements.add(renderMarkdown(returnValue))
-
-        return section().with(elements).withClass("return-value")
-    }
-
-    private fun referredTypeToString(type: ReferredType): String {
-        var representation = type.specifiers.joinToString(" ")
-
-        if (!representation.endsWith("*")) {
-            representation += " "
-        }
-
-        return representation + type.name
-    }
+    fun renderReturnValue(returnValue: ReturnValue) = section(
+        h2("Return Value"), div(
+            h3("Success"), table(
+                tbody(
+                    each(returnValue.success) { child ->
+                        tr(
+                            td(
+                                h4(child.value)
+                            ).withClass("element-name-cell"),
+                            td(
+                                div(
+                                    renderMarkdown(child.description)
+                                ).withClass("element-excerpt-text"),
+                                div(
+                                    "type info comes here"
```

Comment:
Please tell your idea about this section. Should I delete it for now or is this a way to get this data in an easy way?

Reply:
I think, there might be a bit of a misunderstanding going on. Mostly caused by the missing wireframe (which will be agreed on for upcoming UI stuff to prevent such situations from occurring).

The current solutions displays the return type for each of return values (both success and error). This, however, is a bit superfluous, as the type does not change, but solely the value.

Let me give you a complete "wireframe":

![image](https://user-images.githubusercontent.com/11032092/88465152-f8b6e580-cec0-11ea-959b-4e983d686c67.png)

In terms of the `value - description` table, the better source for the CSS might be the submodule table:

https://github.com/underdocs/underdocs/blob/master/underdocs-renderer/src/main/kotlin/underdocs/renderer/output/html/render/section/SectionRenderer.kt#L363

Reply:
Ahh, I was really stupid about this... 😄 sorry

Reply:
No worries! :D As I mentioned earlier, from now on, we will agree on the exact looks of each new rendered section, so such things will not happen.